vnext/ReactUWP/Views/RawTextViewManager.cpp

```diff
@@ -66,13 +64,28 @@ void RawTextViewManager::UpdateProperties(
               textBlock.Text(run.Text());
             }
           }
+
+          NotifyAncestorsTextChanged(instance.operator->(), nodeToUpdate);
         }
       }
     }
   }
   Super::UpdateProperties(nodeToUpdate, reactDiffMap);
 }
 
+void RawTextViewManager::NotifyAncestorsTextChanged(
+    IReactInstance *instance,
+    ShadowNodeBase *nodeToUpdate) {
+  auto host = instance->NativeUIManager()->getHost();
+  ShadowNodeBase *parent = static_cast<ShadowNodeBase *>(
+      host->FindShadowNodeForTag(nodeToUpdate->GetParent()));
+  while (parent) {
+    parent->GetViewManager()->OnDescendantTextPropertyChanged(parent);
```

Comment:
You should cast to TextViewManager here and then try the call to OnDescendantTextPropertyChanged, rather than have all ViewManagers have this function.